docs/iris/src/whatsnew/contributions_2.0/incompatiblechange_2017-Oct-24_netcdf-no-unlim.txt

```diff
@@ -0,0 +1,8 @@
+* Deprecated FUTURE flag :attr:`iris.Future.netcdf_no_unlimited`.
+
+* Removed deprecated behaviour that automatically set the length of the outer netCDF variable to 'UNLIMITED' on save.
```

Comment:
Perhaps change 'con save' to 'when saving in netCDF format' or something similar, to give more context